changelogs/unreleased/gh-7129-improve-check-for-dangerous-select-calls.md

```diff
@@ -1,6 +1,6 @@
 ## feature/box
 
-* Improved check for dangerous select calls: calls with `offset + limit <= 100`
-  are now considered safe (i.e., a warning is not issued); 'ALL', 'GE', 'GT',
-  'LE', 'LT' iterators are now considered dangerous by default even with key
-  present (gh-7129).
+* Improved checking for dangerous `select` calls. The calls with
+  `offset + limit <= 100` are now considered safe, which means a warning is not
```

Comment:
1000 should be here, it is the typo. See https://github.com/tarantool/tarantool/pull/7131#discussion_r875926566.

Reply:
Fixed